experimental/generation/generator/packages/library/templates/standard/dynamiclist.template

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "string",
+    "$dynamiclist": true,
+    "$template": "dynamiclist",
```

Comment:
I suspect we could get rid of $dynamiclist: true and look for the $template being dynamiclist.  Could you try that?